src/app/filename_formatter.h

```diff
@@ -87,6 +88,9 @@ namespace app {
   bool get_frame_info_from_filename_format(
     const std::string& format, int* startFrom, int* width);
 
+  // Types of formats that can be used in the filename formatter. E.g. "{frame}"
+  extern std::array<const char*, 12> path_formats;
```

Comment:
We don't (try to not) define global variables. This could be something like `get_templates_list()` in case is required (I think this is not required for a proper fix).